### DISCUSSION TITLE
What are the requirements for the packager achievement?

### DISCUSSION THREAD

**Discussion by spenserblack** [lowlighter/metrics#1066] [Q&A] [answered]
I've [released a package on a personal repo](https://github.com/spenserblack/actions-msrv/pkgs/container/actions-msrv) and noticed that I don't have the packager achievement. My metrics token already has the `read:package` scope. Is there anything else that's required?

**Comment by lowlighter** [comment, marked as answer]
I think it's the same issue as #1005, the GraphQL API doesn't list published containers from ghcr.io registry (not sure if it's intended from GitHub or not)
The workaround was to use the REST API instead, but it was only patched in the base plugin, not the achivement one

**Reply by spenserblack** [reply]
Ah, that makes sense. Is it possible for achievements to look up organizations? I've published a few packages to an organization namespace, and they haven't counted, either.

**Reply by lowlighter** [reply]
#1074 has been merged and should display the achievement correctly (I tested on your profile and it does show)

As for second question, it may be more tricky I guess and I haven't looked up yet. 

Imho it only makes sense if you're at least maintainer/collaborator on the repo that publish the package at organization levels, like if you're working at a large company it'd be strange to consider packages that were published by other teams that you don't even work with. So the way it could work would be to iterate through fetched organizations, fetch their packages and then check on each if you have write access to them (unless there's a more efficient way) but iirc I think it requires `repo` scope to know rights so it's kind of annoying limitation

But could be nice though

**Reply by spenserblack** [reply]
Ah, you're right, it would take a lot for organizations. Probably makes sense to exclude them, in that case.
